enterprise/frontend/src/metabase-enterprise/tenants/components/TenantsListing/TenantsListing.tsx

```diff
@@ -60,76 +56,70 @@ export const TenantsListing = ({
   }, [searchInputValue, tenants]);
 
   return (
-    <>
-      <AdminPaneLayout
-        headerContent={
-          <Flex justify="space-between" w="100%">
-            <Group>
-              <SearchFilter
-                value={searchInputValue}
-                onChange={setSearchInputValue}
-                placeholder={t`Find a tenant`}
-              />
-              {isAdmin && (
-                <ActiveStatusFilter
-                  status={status}
-                  onStatusChange={onStatusChange}
-                />
-              )}
-            </Group>
-            {isAdmin && (
-              <Button
-                variant="filled"
-                onClick={openNewTenantModal}
-              >{t`New tenant`}</Button>
-            )}
-          </Flex>
-        }
-      >
-        <AdminContentTable columnTitles={[t`Tenant`, t`Slug`, t`Users`]}>
-          {filteredTenants.map((tenant) => (
-            <tr key={tenant.id}>
-              <td>
-                <Flex
-                  component={ForwardRefLink}
-                  align="center"
-                  to={Urls.editTenant(tenant.id)}
-                  className={CS.link}
-                  gap="md"
-                >
-                  <UserAvatar
-                    user={{ first_name: tenant.name }}
-                    bg={tenantIdToColor(tenant.id)}
-                  />
-                  <Box component="span" fw={700} c="brand">
-                    {tenant.name}
-                  </Box>
-                </Flex>
-              </td>
-              <td>{tenant.slug}</td>
-              <td>{tenant.member_count || 0}</td>
-              <Box component="td" ta="end">
-                <ActionsPopover tenant={tenant} />
-              </Box>
-            </tr>
-          ))}
-        </AdminContentTable>
+    <div>
+      <Group w="100%" justify="space-between" mb="lg" gap="md">
```

Comment:
Using `AdminPaneLayout` caused issues with spacing around the search bar as it has arbitrary spacing. Using `Group` fixes this:

<img width="1974" height="880" alt="Image" src="https://github.com/user-attachments/assets/c5da80a7-ec0b-4730-befe-a1ffcc0645d9" />